app/views/curation_concerns/base/_form_state.html.erb

```diff
@@ -2,11 +2,16 @@
   <legend>State</legend>
   <div class="form-group">
```

Comment:
Just concerning that we have to update the checked value postfix conditional based on state for each state there is, which I thought simple form might be able to handle.